src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -1516,7 +1478,7 @@ def merge_kubernetes_configurations(existing_file, addition_file, replace, conte
     if addition is None:
         telemetry.set_exception(exception='Failed to load additional configuration', fault_type=consts.Failed_To_Load_K8s_Configuration_Fault_Type,
                                 summary='failed to load additional configuration from {}'.format(addition_file))
-        raise CLIError('failed to load additional configuration from {}'.format(addition_file))
+        raise FileOperationError('failed to load additional configuration from {}'.format(addition_file))
```

Comment:
This is not user fault. Cannot  use FileOperationError

Reply:
using CLIInternalError now